project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.3.0-RC2
+sbt.version=1.3.0
```

Comment:
Not really a concern, but just curious: is there any downside to our projects having different sbt versions?

Reply:
No downside in terms of technical cross-compatibility, but potential downside of introducing more context-switches to our dev flow if they drift too far apart.